schema/contact-list-contacts.yml

```diff
@@ -0,0 +1,21 @@
+contact_list_contacts:
+  list_name:
+    - string
+    - required
+    - name of the contact list
+    - example: "my_list"
+  list_id:
+    - string|integer
+    - required
+    - external id of the contact list
+    - example: "12345"
+  contact_id_type:
+    - string
+    - required
+    - type of the contact id
+    - example: "MDM_INFLUENCER | MDM_OUTLET | GORKANA_INFLUENCER | GORKANA_OUTLET"
```

Comment:
These values are missing from the example:

`NGC3_INFLUENCER | NGC3_OUTLET | C3_INFLUENCER | C3_OUTLET`

Reply:
Updated @sieglerz